src/runtime/tsconfig.json

```diff
@@ -3,7 +3,7 @@
 	"include": ["."],
 
 	"compilerOptions": {
-		"lib": ["es2015", "dom", "dom.iterable"],
+		"lib": ["es2017", "dom", "dom.iterable"],
```

Comment:
What's changing this do? The build and tests seem to run and pass fine with this reverted to `es2015`. Were you thinking of using some newer built-in globals that you didn't end up using?

Reply:
Oh yea... Let me revert this, it was the Object.entries that requires es2017